### PR TITLE
This request contains the entire design for the home page of thenextweb.com

### PR DESCRIPTION
**Pull request by RyelBanfield**
This request contains the entire design for the home page of thenextweb.com

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @RyelBanfield  & @nganifaith**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] please make sure that your PR has a descriptive description.

Top 3 articles section: 
- [x]  make sure that the images are clickable (links).


Categories section:
- [x] the font for the sub-articles font-size is too small. make sure to enlarge it a little bit.

Latest deals section:

- [x] the original website has discount ribbons. make sure to add them.

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

File: index.html [hunk 1]
```diff
@@ -1,10 +1,855 @@
...
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main class="main-section">
+    <section class="top-section">
+      <div class="feature-article content">
+        <a href="#">
+          <div class="logo-date-wrapper">
+            <img src="./assets/tnw.svg" alt="" class="main-logo" />
```
Comment (umairarshadbutt) [inline index.html]: Add the description in alt attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @nganifaith 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes 👇.
- [x] Add the description in the alt attribute
- [x] MIT license link is not working in the README file

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @RyelBanfield and @nganifaith  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] The screenshot img from the readme file is broken, fix that by providing the correct path to the screenshot of your clone
- [x] The links to your linkedin profiles are broken (both of you) , fix them before submitting for re-review
- [x] The license text you added on the readme file, contains some grammatical errors. Advice: Since you have the `LICENSE` file on the root directory of your project, I highly recommend you to link it on the readme file instead of manually type the license there (on license section)
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:


_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by RyelBanfield** [comment]
Thank you for your reviews.

**Comment by nganifaith** [comment]
Thank you all for your reviews.

**Pull request merged**